### ISSUE TITLE
Spotify RSS Feed

### ISSUE THREAD

**Issue by dragonito** [jgonyea/grav-plugin-podcast#24]
Hi there,

i would like to add the rss feed to Spotify, but Spotify says 'Your RSS feed failed to load. Make sure you added the right link.' Itunes worked well. There are no further information about the problem with Spotify, perhaps you got some ideas to fix it.

I found a guideline:

https://podcasters.spotify.com/terms/Spotify_Podcast_Delivery_Specification_v1.5.pdf

Thanks a lot

Robin

**Comment by dragonito** [comment]
I replaced following

```
<itunes:summary>{{episode.content|striptags|safe_truncate(120)}}</itunes:summary>
```

to

```
<itunes:summary>{{episode.content|striptags|truncate(120, true, " ", "&#x2026;")}}</itunes:summary>
```

works :D

**Comment by jgonyea** [comment]
Hm, what's the difference that the safe truncate isn't doing that the truncate is? Could you send me a sample episode page Markdown?

**Comment by jgonyea** [comment]
Can you also provide what the before/ after result of the rss was? Thanks

**Comment by dragonito** [comment]
I use ```&#x2026;``` instead of ```&hellip;```, rss-xml doesnt support hellip and i got errors about it from Spotify. You could cdata the content or use an other variant like ```&#x2026;``` 

I could post you an example of it, when i am @ my pc. I you like i could fix it with an mr.

**Comment by jgonyea** [comment]
I'll have this fixed in the next release.  Thanks.

**Comment by dragonito** [comment]
Nice :D thx a lot

**Comment by jgonyea** [comment]
Addressed in ver 2.1.5
